lib/utils/system_bars.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+/// System bars styling is set
+// The AppBar & SliverAppBar widgets will automatically set the status bar
+// brightness when they are rendered. If there is no Nav/AppBar on the first
+// page, then you can get a similar effect using the AnnotatedRegion widget
+//
+// If you have an AppBar or SliverAppBar then it will interfere with your
+// calls to SystemChrome.setSystemUIOverlayStyle and you can instead just
+// change the brightness property on your AppBar.
+SystemUiOverlayStyle systemBarStyle(BuildContext context) {
+  // Copy the default style and then modify key values
+  final lightStyle1 = SystemUiOverlayStyle.dark.copyWith(
+    // Status bar - iOS and Android
+    statusBarBrightness: Brightness.light, // iOS only
+    statusBarColor: Colors.transparent, // Android v6 and above
```

Comment:
what does `Android v6 and above` mean when we only support SDK21 and above?

Reply:
It means this property is only effective on Android 6 and above. So doco exact quote is:

'Only honored in Android version M and greater'

Added these comments otherwise anyone reading the code will get very confused very fast as some properties iOS only, some only specific Android versions. I'll add comments above the line and quote dodo.

Reply:
Its a pain because they are almost all different. So have added better comments above lines. 👍

Reply:
Ah ok in that case better to use Android version letters or even better the SDK number, for Android devs bit confusing to see Android version numbers not letter names.

Reply:
Have letter and API level now in comments 👍